helm-frontend/src/components/LeaderboardTables.tsx

```diff
@@ -282,20 +269,14 @@ export default function LeaderboardTables({
                         className={`${
                           cellIdx === 0 ? "text-lg sticky left-0" : ""
                         } ${idx % 2 === 0 ? "bg-gray-50" : "bg-white"}`}
-                        title={`Click value to see predictions for ${getGroupForRunName(
-                          getHeaderValue(activeGroupsTable.header[cellIdx]),
-                        )}: ${getModelForRunName(String(row[0].value))}`}
+                        title={`Click value to see predictions for ${getHeaderValue(
+                          activeGroupsTable.header[cellIdx],
+                        )}: ${String(row[0].value)}`}
```

Comment:
For accessibility reasons, the `title` attribute should be set on the `<a>` element instead of on the `<td>` - that is, you should pass it into `<Link>` / `<ReactRouterLink>`.

Okay to do this in a new PR if it is too complicated.